packages/auth/src/platform_browser/recaptcha/recaptcha.ts

```diff
@@ -78,20 +82,61 @@ export class RecaptchaConfig {
   siteKey: string = '';
 
   /**
-   * The reCAPTCHA enablement status of the {@link EmailAuthProvider} for the current tenant.
+   * The list of providers and their enablement status for reCAPTCHA.
```

Comment:
nit: reCAPTCHA Enterprise.

Since we have multiple recaptcha tokens supported in the SDK, let's call out rCE wherever possible.